misk/src/test/kotlin/misk/client/MockWebServerService.kt

```diff
@@ -16,7 +16,9 @@ class MockWebServerService(val unixSocketFile: String?) : AbstractIdleService()
 
     server = MockWebServer()
     unixSocketFile?.let {
-      server!!.setServerSocketFactory(UnixDomainServerSocketFactory(file))
+      server!!.run {
```

Comment:
you can probably fold the `run` call

Reply:
(it’s caused by a limitation of the fancy deprecation feature)

Reply:
fixed